### ISSUE TITLE
Application fails at allowing login of multiple users with their fingerprints on the same device.

### ISSUE THREAD

**Issue by BautistaDalmasso** [BautistaDalmasso/Skynet-LCDS-2024#26]
How to replicate:
* Register or login with "user a", make sure that your fingerprint is registered on that device for that user. Then log out.
* Register or login with "user b", make sure that your fingerprint is registered on that device for that user. Then log out.
* Try to log in with "user a" using fingerprint authentication, **this will fail**.

If you wanted to login with "user b" using fingerprint authentication it would work as expected.

TLDR: Fingerprint authentication can only remember the latest user to have registered their fingerprints in that given device.
